.github/workflows/ci.yml

```diff
@@ -5,20 +5,19 @@ jobs:
     strategy:
       matrix:
         pg:
-          - 11
-          - 12
           - 13
           - 14
           - 15
           - 16
+          - 17
         ruby:
-          - "3.1"
```

Comment:
> Remove builds for Ruby 3.1 (Rails 8 only supports 3.2+)

The other versions of Rails still support 3.1 but it was much easier to just remove it completely instead of special-casing Rails 8. I _think_ this should be fine as Ruby 3.1 is EOL in 4 months?